etna/pipeline/base.py

```diff
@@ -24,6 +25,87 @@
 from etna.metrics import Metric
 from etna.metrics import MetricAggregationMode
 
+Timestamp = Union[str, pd.Timestamp]
+
+
+class CrossValidationMode(Enum):
+    """Enum for different cross-validation modes."""
+
+    expand = "expand"
+    constant = "constant"
+
+
+class FoldMask(BaseMixin):
+    """FoldMask class for masks."""
```

Comment:
Please, add a more precise description of this class. It is really not obvious what it does by this small line.

Reply:
Fixed, got the this description from etna-utils)